jobbergate_cli/jobbergate_api_wrapper.py

```diff
@@ -624,9 +617,9 @@ def create_job_script(
         files = {"upload_file": open(param_filename, "rb")}
 
         # Possibly overwrite script name
-        job_script_name_from_param = param_dict["jobbergate_config"]["job_script_name"]
+        job_script_name_from_param = param_dict["jobbergate_config"]["job_name"]
         if (
-            "job_script_name" in param_dict["jobbergate_config"]
+            "job_name" in param_dict["jobbergate_config"]
             and job_script_name_from_param != ""
```

Comment:
Silly question: why break the line after the `if`?

Reply:
We are using black in this project, it is the way it does